From ac9d7d230effdb97870c2f317b3b45449bde1769 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <am.devel@gmail.com>
Date: Wed, 11 Apr 2018 15:40:02 +0800
Subject: [PATCH] validate: fix template paths when buildir is different

Compilation just fails otherwise because it's not able to locate the files.

Upstream-Status: Submitted [https://bugzilla.gnome.org/show_bug.cgi?id=795149]
Signed-off-by: Anuj Mittal <am.devel@gmail.com>

---
 gst/validate/Makefile.am | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/gst/validate/Makefile.am b/gst/validate/Makefile.am
index 342730a..3fcc3e5 100644
--- a/gst/validate/Makefile.am
+++ b/gst/validate/Makefile.am
@@ -48,14 +48,17 @@ source_h = \
 nodist_libgstvalidate_@GST_API_VERSION@_la_SOURCES = $(built_source_make)
 nodist_libgstvalidate_@GST_API_VERSION@include_HEADERS = $(built_header_make)
 
+template_h = $(patsubst %,$(srcdir)/%, gst-validate-enum-types.h.template)
+template_c = $(patsubst %,$(srcdir)/%, gst-validate-enum-types.c.template)
+
 gst-validate-enum-types.h: $(source_h)
 	$(AM_V_GEN)$(GLIB_MKENUMS) \
-	--template gst-validate-enum-types.h.template \
+	--template $(template_h) \
 	$^ > gst-validate-enum-types.h
 
 gst-validate-enum-types.c: $(source_h)
 	$(AM_V_GEN)$(GLIB_MKENUMS) \
-	--template gst-validate-enum-types.c.template \
+	--template $(template_c) \
 	$^ > gst-validate-enum-types.c
 
 EXTRA_DIST= \
-- 
2.7.4

